openhcl/underhill_core/src/servicing.rs

```diff
@@ -79,6 +79,9 @@ mod state {
         /// Shared pool information.
         #[mesh(10001)]
         pub shared_pool_state: Option<page_pool_alloc::save_restore::PagePoolState>,
+        /// Private pool information.
+        #[mesh(10002)]
```

Comment:
meta: how are we tracking these instances of "unstable" mesh numbering, and ensuring they don't leak into release builds?

Reply:
I'm not sure we have anything in particular. I think we need to be vigilant/add it to our release checklist though. 

Or perhaps we should take the stance that these must have corresponding GH issues in the comment?

Reply:
#460 for now